docs/data/date-pickers/base-concepts/ComponentFamilies.js

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import Link from 'next/link';
```

Comment:
In theory, we would need to import from https://github.com/mui/material-ui/blob/master/docs/src/modules/components/Link.tsx to support multi language, but since we are not actively supporting Chinese, it's transparent for now.

Reply:
It will be removed in a followup PR #10661